packages/babel-cli/bin/babel-node.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require("../lib/babel-node");
+require("babel-node");
```

Comment:
The idea in this case is that the `babel-node` package will provide its own command https://github.com/babel/babel/blob/master/packages/babel-node/package.json#L32 so we'd want to delete this whole file.

Reply:
Makes sense to me. However I made the changes according to @hzoo comment [here](https://github.com/babel/babel/pull/6251#discussion_r139776319). I'll remove the file and update the PR.

Reply:
Oh gotcha. I think the issue is that as far as I know `npm` doesn't handle two packages providing the same command very well. When we had the `babel` and `babel-cli` packages, it confused a lot of people because they couldn't just uninstall `babel`, they also have to explicitly install `babel-cli` _after_. If you're removing this from `babel-cli`, then I think we need to remove all references to it entirely.

Reply:
I think you got confused regarding deleting the `index.js` file from `babel-node`? This is the file I introduced in `babel-node` package as currently there is no default export. Check `master` branch https://github.com/babel/babel/tree/master/packages/babel-node

I introduced this so that `babel-node` package can be imported if required in future. If we never intend to import `babel-node` in future then there is no need of this file and I'll just remove it from my PR. Please let me know your thoughts on this.